File: NQT/Arrays/Python/arrangement.py
# arrange given array in ascending order and descending order

def main():
    print("Enter the elements of the array separated by spaces:")
    user_input = input()
    main_array = list(map(int, user_input.split()))
    ascending, descending = arrange_array(main_array)
    print("Original array:", main_array)
    print("Ascending order:", ascending)
    print("Descending order:", descending)


def arrange_array(arr):
    # Sort the array in ascending order
    ascending_order = sorted(arr)
    
    # Sort the array in descending order
    descending_order = sorted(arr, reverse=True)
    
    return ascending_order, descending_order

if __name__ == "__main__":
    main()

File: NQT/Arrays/Java/Arrangement.java
import java.util.*;
public class Arrangement {
    // arrange the elements in array in both ascending and descending order
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the size of the array:");
        int array_size = scanner.nextInt();
        System.out.println("Enter " + array_size + " integers:");
        int arrangement_list[] = new int[array_size];
        for(int i = 0;i<array_size;i++){
            arrangement_list[i] = scanner.nextInt();
        }
        scanner.close();
        int[] ascending = ascending(arrangement_list, array_size);
        int[] descending = descending(arrangement_list, array_size);
        System.out.println("Original array: " + Arrays.toString(arrangement_list));
        System.out.println("Ascending order: " + Arrays.toString(ascending));
        System.out.println("Descending order: " + Arrays.toString(descending));
    }
    private static int[] ascending(int array[], int array_size){
        Arrays.sort(array);
        return array;
    }
    private static int[] descending(int array[], int array_size){
        Arrays.sort(array);
        int descending_array[] = new int[array_size];
        for(int i = 0;i<array_size;i++){
            descending_array[i] = array[array_size - 1 - i];
        }
        return descending_array;
    }
}
